Bio/KEGG/KGML/KGML_parser.py

```diff
@@ -80,7 +80,109 @@ def parse(handle, debug=0):
 
 
 class KGMLParser(object):
-    """Parses a KGML XML Pathway entry into a Pathway object."""
+    """Parses a KGML XML Pathway entry into a Pathway object.
+        Example: Read and parse large metabolism file
+
+    >>> from Bio.KEGG.KGML.KGML_parser import read
+    >>> pathway = read(open('KEGG/ko01100.xml', 'rU'))
```

Comment:
Does everything work without the universal new line mode option which has been deprecated? See https://bugs.python.org/issue15204